README.md

```diff
@@ -1,2 +1,36 @@
+![Github CI/CD](https://github.com/rsksmart/pmt-builder/actions/workflows/workflow.yml/badge.svg)
+
 # pmt-builder
-PartialMerkleTree builder lib
+
+PartialMerkleTree builder library.
+
+This library can be used to build a partial merkle tree for a block of transactions. A partial merkle tree is a pruned version of a full merkle tree, containing only the parts needed to prove a given transaction is part of the tree. 
+
+## Installation
+
+> npm install @rsksmart/pmt-builder
+
+### Install dependencies
+
+> npm install
+
+## Usage
+
+To build the tree, simply call the `buildPMT` function and pass the `leaves` as a list of strings and a `filteredHash` in string format, i.e buildPMT(leaves: string[], filteredHash: string)
+
+First parameter {leaves}: An array of transaction hash of all transactions in a block. 
+Second parameter {filteredHash}: transaction hash (the hash without witness) of the coinbase transaction.
```

Comment:
It doesn't have to be the coinbase transaction. The PMT can be built for any transaction

Reply:
Resolved